src/RtcDS3231.h

```diff
@@ -344,7 +349,7 @@ template<class T_WIRE_METHOD> class RtcDS3231
         }
 
         // Temperature is represented as a 10-bit code with a resolution
-        // of 1/4th �C and is accessable as a signed 16-bit integer at
+        // of 1/4th �C and is accessable as a signed 16-bit integer at
```

Comment:
looks like your editor is doing extra work here and removing the degree symbol.
Please correct.